static/js/actions/course_enrollments.js

```diff
@@ -33,3 +36,16 @@ export const addCourseEnrollment = (courseId: string): Dispatcher<*> => {
       });
   };
 };
+
+export const showEnrollPayLaterSuccessMessage = (courseId: string): Dispatcher<*> => {
+  return (dispatch: Dispatch) => {
+    dispatch(showEnrollPayLaterSuccess({course_id: courseId}));
```

Comment:
If the only thing we care about is the course key, can we just pass `courseId` here by itself instead of within an object?